compiler/luci/import/include/luci/Import/CircleReader.h

```diff
@@ -120,6 +127,20 @@ class CircleReader
   circle::BuiltinOperator builtin_code(const circle::OperatorT &op) const;
   std::string opcode_name(const circle::OperatorT &op) const;
 
+public: // direct API
+  CircleOperatorCodes native_opcodes() const { return wrap(_native_model->operator_codes()); }
+  CircleBuffers native_buffers() const { return wrap(_native_model->buffers()); }
+  CircleTensors native_tensors() const { return wrap(_native_subgraph->tensors()); }
+  CircleOperators native_operators() const { return wrap(_native_subgraph->operators()); }
+  VectorWrapper<int32_t> native_inputs() const { return wrap(_native_subgraph->inputs()); }
+  VectorWrapper<int32_t> native_outputs() const { return wrap(_native_subgraph->outputs()); }
+  std::string native_name() const { return _native_subgraph->name()->str(); }
+  circle::DataFormat native_data_format() const { return _native_subgraph->data_format(); }
+  CircleMetadataSet native_metadata() const { return wrap(_native_model->metadata()); }
+
+  circle::BuiltinOperator builtin_code(const circle::Operator *op) const;
+  std::string opcode_name(const circle::Operator *op) const;
```

Comment:
Ahh, I forgot to add their definitions.

Reply:
Hmm, let's add definitions in another PR since we need introduce finctions without UnPack first.